core/arch/arm/plat-imx/drivers/imx_csu.c

```diff
@@ -103,6 +103,9 @@ static TEE_Result csu_init(void)
 	const struct csu_config *csu_config = NULL;
 	const struct csu_setting *csu_setting = NULL;
 
+	if (IS_ENABLED(CFG_IMX_INSECURE_CSU_SETTING))
+		IMSG("Device running with an insecure setting, consult the security reference manual from NXP");
```

Comment:
I believe we allow long debug print lines since a while back?

Reply:
May I suggest something like:
```IMSG("Device running with an open security settings, please refer to OP-TEE i.MX platform documentation");``` ?
OPTEE i.MX documentation being the readthedocs page where NXP will add the disclaimer.